pathways/inspections.py

```diff
@@ -234,29 +226,82 @@ def compute_max_inspectable_stems(num_stems, stems_per_box, within_box_pct):
     return max_stems
 
 
-def inspect(config, shipment, n_boxes_to_inspect):
-    """Select boxes from shipment based on specified selection strategy.
-    Inspect selected boxes using both end strategies (to detection, to completion)
+def inspect(config, shipment, n_units_to_inspect):
+    """Select units from shipment based on specified selection strategy.
+    Inspect selected units using both end strategies (to detection, to completion)
     Return number of boxes opened, stems inspected, and infested stems found for
     each end strategy.
 
     :param config: Configuration to be used
     :param shipment: Shipment to be inspected
-    :param n_boxes_to_inspect: Number of boxes to inspect defined by sample functions.
+    :param n_units_to_inspect: Number of units to inspect defined by sample functions.
     """
+    # Disabling warnings, possible future TODO is splitting this function.
+    # pylint: disable=too-many-locals,too-many-branches,too-many-statements
+
+    unit = config["inspection"]["unit"]
     pathway = shipment["pathway"]
     stems_per_box = config["stems_per_box"]
     stems_per_box = get_stems_per_box(stems_per_box, pathway)
+    num_stems = shipment["num_stems"]
     num_boxes = shipment["num_boxes"]
```

Comment:
With b8828888931a10656f1246aa89e9330a5ef03625 (#66) you can now access values in shipment using attribute access syntax, i.e., `shipment.num_stems` instead of `shipment["num_stems"]`. That still results in long, sometimes less readable equations (then those with individual variables), but it may help with having a lot of variables defined in one function. (The idea is that the variables are grouped under one, but the access syntax is simpler [than a dictionary] and a little bit more readable when used in equations, but it is always a trade-off.)

Reply:
...but you would have to merge the master branch to your branch again.